packages/python/accern_data/accern_data.py

```diff
@@ -303,26 +364,27 @@ def validate_filters(filters: Optional[FiltersType]) -> FiltersType:
         if filters is None:
             return {}
         for key in filters.keys():
-            if key not in FilterField:
+            if key not in FILTER_FIELD:
```

Comment:
no, we have an exclusion on filters
for example harvested_at cannot be a filter or date everything else is fine

Reply:
even wrong ones? like `foo`?

Reply:
yes